packages/cli/src/commands/generate/service/templates/service.ts.template

```diff
@@ -31,6 +31,8 @@ export const delete${singularPascalName}: MutationResolvers['delete${singularPas
   })
 }<% } %><% if (relations.length) { %>
 
-export const ${singularPascalName}: Partial<${singularPascalName}Resolvers>  = {<% relations.forEach(relation => { %>
-  ${relation}: (_obj, gqlArgs) => db.${singularCamelName}.findUnique({ where: { id: gqlArgs?.root?.id } }).${relation}(),<% }) %>
+export const ${singularPascalName}: Partial<${singularPascalName}RelationResolvers>  = {<% relations.forEach(relation => { %>
```

Comment:
Since we're generating the new type here, perhaps we could remove the Partial and make all the keys optional? I only had to add Partial because of the `avoidOptionals` config which we need for strict mode for calling the resolvers in tests (see https://github.com/redwoodjs/redwood/pull/6222#issuecomment-1230156868)

Since RelationResolvers is a new type, maybe the optionality won't hurt elsewhere!

Reply:
Yes, we could definitely do this!

Reply:
That was super easy!